test/integration/lib/test-actions/index.ts

```diff
@@ -4,6 +4,7 @@ import { parseCookies } from '../utils'
 import { WalletAddress, PendingGrant } from '@interledger/open-payments'
 import { AdminActions, createAdminActions } from './admin'
 import { OpenPaymentsActions, createOpenPaymentsActions } from './open-payments'
+import { urlWithoutTenantId } from '../../../../packages/backend/src/shared/utils'
```

Comment:
I'm surprised we can reach across to the other package like this. I think we should just re-declare/copy the function somewhere in this project just to make sure it's ok.

Reply:
I thought it was pretty neat. :-) 
Fixed it now.